frontend/public/components/graphs/area.tsx

```diff
@@ -136,8 +137,9 @@ export const AreaChart: React.FC<AreaChartProps> = ({
   return (
     <PrometheusGraph className={className} ref={containerRef} title={title}>
       {processedData?.length ? (
-        <PrometheusGraphLink query={query} ariaChartLabel={ariaChartLabel}>
+        <PrometheusGraphLink query={query} ariaChartLinkLabel={ariaChartLinkLabel}>
           <Chart
+            ariaTitle={title || ariaChartTitle}
```

Comment:
Should `ariaChartTitle` take precedence if both are defined?

Reply:
fixed